modules/local/combinegvcfs.nf

```diff
@@ -0,0 +1,65 @@
+process COMBINEGVCFS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
```

Comment:
You might want to copy the environment.yml from the original module in this repo.

Reply:
I did so. Following nf-core procedures, I moved the module to its own directory in modules/local/combine_gvcfs and renamed the nf file to main.nf for "includes".